modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/IborIndexData.csv

```diff
@@ -1,284 +1,298 @@
+JPY-LIBOR-6M,JPY,TRUE,Act/360,GBLO,2,GBLO,GBLO+JPTO,6M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-7M,JPY,FALSE,Act/360,GBLO,2,GBLO,GBLO+JPTO,7M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-8M,JPY,FALSE,Act/360,GBLO,2,GBLO,GBLO+JPTO,8M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-9M,JPY,FALSE,Act/360,GBLO,2,GBLO,GBLO+JPTO,9M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-10M,JPY,FALSE,Act/360,GBLO,2,GBLO,GBLO+JPTO,10M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-11M,JPY,FALSE,Act/360,GBLO,2,GBLO,GBLO+JPTO,11M,LastBusinessDay,11:55,Europe/London,Act/365F
+JPY-LIBOR-12M,JPY,TRUE,Act/360,GBLO,2,GBLO,GBLO+JPTO,12M,LastBusinessDay,11:55,Europe/London,Act/365F
+,,,,,,,,,,,,
+JPY-TORF-1M,JPY,TRUE,Act/365F,JPTO,2,JPTO,JPTO,1M,LastBusinessDay,15:00,Asia/Tokyo,Act/365F
+JPY-TORF-3M,JPY,TRUE,Act/365F,JPTO,2,JPTO,JPTO,3M,LastBusinessDay,15:00,Asia/Tokyo,Act/365F
+JPY-TORF-6M,JPY,TRUE,Act/365F,JPTO,2,JPTO,JPTO,6M,LastBusinessDay,15:00,Asia/Tokyo,Act/365F
+,,,,,,,,,,,,
+USD-LIBOR-1W,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,1W,Following,11:55,Europe/London,Act/360
+USD-LIBOR-2W,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,2W,Following,11:55,Europe/London,Act/360
+USD-LIBOR-1M,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,1M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-2M,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,2M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-3M,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,3M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-4M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,4M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-5M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,5M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-6M,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,6M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-7M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,7M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-8M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,8M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-9M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,9M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-10M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,10M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-11M,USD,FALSE,Act/360,GBLO,2,GBLO,GBLO+USNY,11M,LastBusinessDay,11:55,Europe/London,Act/360
+USD-LIBOR-12M,USD,TRUE,Act/360,GBLO,2,GBLO,GBLO+USNY,12M,LastBusinessDay,11:55,Europe/London,Act/360
+,,,,,,,,,,,,
+USD-BSBY-1M,USD,TRUE,Act/360,USGS,2,USGS,USGS,1M,LastBusinessDay,08:00,America/New_York,Act/360
+USD-BSBY-3M,USD,TRUE,Act/360,USGS,2,USGS,USGS,3M,LastBusinessDay,08:00,America/New_York,Act/360
+USD-BSBY-6M,USD,TRUE,Act/360,USGS,2,USGS,USGS,6M,LastBusinessDay,08:00,America/New_York,Act/360
+USD-BSBY-12M,USD,TRUE,Act/360,USGS,2,USGS,USGS,12M,LastBusinessDay,08:00,America/New_York,Act/360
+,,,,,,,,,,,,
+USD-TERMSOFR-1M,USD,TRUE,Act/360,USGS,2,USGS,USGS,1M,LastBusinessDay,05:00,America/New_York,Act/360

```

Comment:
We have `AMERIBORTERM` but `TERMSOFR` and `TERMSONIA`. If these are all similar, we should name similarly, maybe changing to `TERMAMERIBOR`?

Reply:
The word "term" is used with different meaning in those two contexts. AMERIBORTERM is a term deposit rate while TERMSOFR is a term derivative rate from Overnight SOFR-linked derivatives. 
We can harmonize the names, but we still have the issues reported below, that the overnight-linked term rates need also the term rate provider in the name to avoid confusion when several providers publish similar rates.